### PR TITLE
feat: client side redirects

### PR DESCRIPTION
**Pull request by pmelab**
See: https://github.com/dai-shi/waku/discussions/929

### CODE REVIEW

**Comment by dai-shi** [comment]
Would you open a new discussion to discuss possible approaches?

**Comment by pmelab** [comment]
> Would you open a new discussion to discuss possible approaches?

Done!

File: README.md [hunk 1]
```diff
@@ -545,6 +545,88 @@ export const Component = () => {
 };
 ```
 
+### Redirects
```
Comment (dai-shi) [inline README.md:548]: We carefully design and write our README (which will be on the website).
Can you write it down in docs instead for now?

Comment (pmelab) [reply README.md]: Sure! There you go.

File: docs/redirects.mdx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+slug: redirects
+title: Redirects
+description: How to add HTTP redirects to your Waku project.
+---
+
+### Redirects
+
+Redirects are not handled by Waku directly. Instead, you can use either a custom middleware or the hosting environment to achieve that. The `<Link />` component does not deal with redirects either and will by default show the **404** page instead. To resolve this, you have to add an additional redirect for each redirected path, that points Waku to the correct `RSC` file. If there is a redirect from `/old` to `/new`, there also has to be one from `/RSC/old.txt`` to `/RSC/new.txt`to make the `<Link />` component`s smooth page transition work.
+
+> The `/RSC/` file naming convention is [subject to change](https://github.com/dai-shi/waku/discussions/929#discussioncomment-10825975) in future versions of Waku.
```
Comment (dai-shi) [inline docs/redirects.mdx:11]: I'm going to change it in the next _patch_ version. 😝

**Review by dai-shi — APPROVED** [review]
LGTM!

**Pull request merged**